README.md

```diff
@@ -20,78 +23,67 @@ query them based on their location.
-- Functor/Mondad (FP through [cats](https://typelevel.org/cats/)).
-- The real world/side effects using the [IO Monad](http://book.realworldhaskell.org/read/io.html).
-- Testing
-- [Types and type clases](http://learnyouahaskell.com/types-and-typeclasses)
-- [Implicits](https://docs.scala-lang.org/tour/implicit-parameters.html)
-- [hlists](https://www.scala-exercises.org/shapeless/heterogenous_lists)
+    - [Functional Programming HOWTO](https://docs.python.org/es/3/howto/functional.html#functional-programming-howto)
 
 ## Books
-- [https://underscore.io/books](https://underscore.io/books/)
+- Coming soon
 
 ## Videos
-- [Let’s Code Real World App Using Purely Functional Techniques (in Scala)](https://youtu.be/m40YOZr1nxQ)
+- Coming soon
 
 ## Tools
-- [Scala](https://www.scala-lang.org/2020/06/29/one-click-install.html)
-- [IntelliJ (IDE)](https://www.jetbrains.com/idea/download/)
-- [sbt](https://www.scala-sbt.org/)
+- [Python]https://www.python.org/)
+- [Code](https://code.visualstudio.com/)
 
 ## Libraries
-- [cats](https://typelevel.org/cats/): Library for FP.
-- [cats-effect](https://typelevel.org/cats-effect/): IO Monad in Scala.
-- [FS2](https://fs2.io/index.html): Functional streams.
-- [Doobie](https://tpolecat.github.io/doobie/): Functional layer for JDBC.
-- [Sangria](https://sangria-graphql.github.io/): Scala library for GraphQL.
-- [ScalaTest](https://www.scalatest.org/): Scala testing library.
-- [ScalaCheck](https://www.scalacheck.org/): Library for random testing of program properties inspired by [QuickCheck](https://hackage.haskell.org/package/QuickCheck).
-- [http4s](https://http4s.org/): Library fot HTTP
+- [Django](https://docs.djangoproject.com): Library for FP.
+- [Geospatial libraries](https://docs.djangoproject.com/en/3.2/ref/contrib/gis/install/geolibs/)
+- [GeoDjango Model API](https://docs.djangoproject.com/en/3.2/ref/contrib/gis/model-api/)
+- [Testing in Django](https://docs.djangoproject.com/en/3.2/topics/testing/): Python testing library.
 
 ## Exercises
-- [Std lib](https://www.scala-exercises.org/std_lib/asserts)
-- [Fp in Scala](https://www.scala-exercises.org/fp_in_scala/getting_started_with_functional_programming)
-- [Cats](https://www.scala-exercises.org/cats/semigroup)
-- [Circe](https://www.scala-exercises.org/circe/Json)
-- [Doobie](https://www.scala-exercises.org/doobie/connecting_to_database)
-- [ScalaCheck](https://www.scala-exercises.org/scalacheck/properties)
+
 
 ## Basic commands
-SBT console
+Django Shell
 ```
-$ sbt
+$ python manage.py shell
 ```
 
-Running sbt commands inside the SBT console
+Running test
 ```
-// sbt console
-$ sbt
+// running all tests
+$ python manage.py test
+
+// running api test
+$ python manage.py test api.test
+
+// Creates new migration(s) for api app
+$ python manage.py makemigrations api
+
+// Updates database schema
+$ python manage.py migrate api
+
+// Create virtual environment
+$ python -m venv env
+
+// Activate Virtual Environment
+$ source mypython/bin/activate
```

Comment:
The name of virtual environment is not the same

Reply:
@jmercadoc Better use docker instead of using venv locally, this can help to change local environments

Reply:
ok, I'll switch to docker